### ISSUE TITLE
Update hero

### ISSUE THREAD

**Issue by lpsinger** [nasa-gcn/gcn.nasa.gov#372]
Use this graphic, which has a nice blank star field on the left of the image where the text can go.

![201754_GNC_Network_Diagram_v9](https://user-images.githubusercontent.com/728407/199047245-c2a5c9ed-7db8-4f1b-943c-e50a2720e9bf.jpg)

**Comment by dakota002** [comment]
@lpsinger is this what you are looking for, 
![image](https://user-images.githubusercontent.com/11023698/199310499-0f80b3fc-600e-4de7-abb0-ff4247ed6ea9.png)

Or are you thinking more of a full page-width image?

**Comment by lpsinger** [comment]
Yes, page width. Also see if we can get rid of the flat colored background under the text.

**Comment by dakota002** [comment]
@lpsinger what are your thoughts on this?

![image](https://user-images.githubusercontent.com/11023698/199503738-ec3645c2-c894-4b9c-8ed2-bce8ed8578db.png)

**Comment by lpsinger** [comment]
That looks really nice.

Can you show a screen shot that includes a bit of the page content too? My concern is that the image may look misaligned given how far it extends to the right.

**Comment by dakota002** [comment]
Yea, just need a sec, trying to resolve some of the sizing issues with mobile display too

**Comment by dakota002** [comment]
Here is with some of the lower content, I also changed some of the padding on the sides of the text to get it aligned with the rest of the content
![image](https://user-images.githubusercontent.com/11023698/199521954-78f34b15-f49c-4998-b74e-20dd033969ee.png)

**Comment by dakota002** [comment]
Full desktop view is as previous, here are some of the smaller rendered versions.  @lpsinger  what do you think?

Laptop:

![image](https://user-images.githubusercontent.com/11023698/199569372-f1b416b5-fef4-417d-9b6e-a5b42383a7d8.png)

Tablet:

![image](https://user-images.githubusercontent.com/11023698/199569527-79e21d8d-c337-44ae-81c6-47a538e286ec.png)


Mobile-L:

![image](https://user-images.githubusercontent.com/11023698/199569600-ac3a2480-44de-4ce0-9d98-c751bfc9b996.png)

Mobile-S:

![image](https://user-images.githubusercontent.com/11023698/199569944-0b15f95e-c81d-4f63-b646-f1d2817b487a.png)

**Comment by lpsinger** [comment]
What can we do so that the image does not get cut off on the top? In the first of the screen shots above, the "Transient" label in the image is cut off.

**Comment by dakota002** [comment]
@lpsinger are you talking about the Laptop sizing? I was just scrolled down a little

Here is the full top view:
![image](https://user-images.githubusercontent.com/11023698/200045976-ed0aa3df-3c16-4d37-ae42-a03ac7a2bdb4.png)
